nucleus/deployment/common/src/main/java/com/sun/enterprise/deployment/deploy/shared/InputJarArchive.java

```diff
@@ -311,20 +310,13 @@ protected static JarFile getJarFile(URI uri) throws IOException {
         JarFile jf = null;
         try {
             File file = new File(uri);
-            if (file.exists()) {
-                jf = new JarFile(file);
-            }
+            jf = new JarFile(file);
```

Comment:
Does the removal of the existance check save the time?

Reply:
Yes, since the jar will exist, unless deleted by the user while deployment is ongoing, this is unlikely and so if this happens then the exception is better.